src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -98,7 +100,28 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-        return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, Messages.format(editedPerson)));
+        long clashes = model.checkClashes(editedPerson);
+        List<Person> clashingPersons = model.getClashingPersons(editedPerson);
+        if (clashes == 0) {
+            return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, Messages.format(editedPerson)));
+        } else {
+            return new CommandResult(
+                    String.format(MESSAGE_EDIT_PERSON_SUCCESS, Messages.format(editedPerson))
+                    + String.format(
+                            MESSAGE_HAS_CLASHES,
+                            clashes,
+                            Messages.listFormat(
+                                    clashingPersons,
+                                    person -> String.format(
+                                            "Name: %s | Schedule: %s\n",
+                                            person.getName(),
+                                            person.getSchedule()
```

Comment:
Feels like there is too many level of indentation.

Reply:
I abstracted the generating of the error message to a function in Messages.java